README.md

```diff
@@ -1,3 +1,8 @@
+> **DEPRECATED!**  
```

Comment:
I think we should make this a H1 header:

```suggestion
> # DEPRECATED! #
```

That way it displays in giant, bold font.